test/access/AccessControl.behavior.js

```diff
@@ -210,8 +211,293 @@ function shouldBehaveLikeAccessControlEnumerable(errorPrefix, admin, authorized,
+
+      await this.accessControl.beginDefaultAdminTransfer(correctPendingDefaultAdmin, { from: defaultAdmin });
+    });
+
+    describe('caller is pending default admin and delayed until is met', async function () {
+      let from;
+
+      beforeEach(async function () {
+        await time.increaseTo(correctIncreaseTo);
+        from = correctPendingDefaultAdmin;
+      });
+
+      it('accepts a transfer and changes default admin', async function () {
+        await this.accessControl.acceptDefaultAdminTransfer({ from });
+        expect(await this.accessControl.hasRole(DEFAULT_ADMIN_ROLE, defaultAdmin)).to.be.false;
+        expect(await this.accessControl.hasRole(DEFAULT_ADMIN_ROLE, newDefaultAdmin)).to.be.true;
+        expect(await this.accessControl.owner()).to.equal(newDefaultAdmin);
+      });
+
+      it('accepts a transfer and emit events', async function () {
+        const receipt = await this.accessControl.acceptDefaultAdminTransfer({ from });
+        expectEvent(receipt, 'RoleRevoked', {
+          role: DEFAULT_ADMIN_ROLE,
+          account: defaultAdmin,
+        });
+        expectEvent(receipt, 'RoleGranted', {
+          role: DEFAULT_ADMIN_ROLE,
+          account: newDefaultAdmin,
+        });
+      });
+
+      it('accepts a transfer resetting pending default admin and delayed until', async function () {
+        await this.accessControl.acceptDefaultAdminTransfer({ from });
+        expect(await this.accessControl.defaultAdminTransferDelayedUntil()).to.be.bignumber.equal(web3.utils.toBN(0));
+        expect(await this.accessControl.pendingDefaultAdmin()).to.equal(ZERO_ADDRESS);
+      });
+    });
+
+    it('should revert if caller is not pending default admin', async function () {
+      await time.increaseTo(correctIncreaseTo);
+      await expectRevert(
+        this.accessControl.acceptDefaultAdminTransfer({ from: other }),
+        `${errorPrefix}: can't accept defaultAdmin`,
+      );
+    });
+
+    describe('delayedUntil not met', async function () {
+      let incorrectIncreaseTo;
+
+      beforeEach(async function () {
+        incorrectIncreaseTo = correctIncreaseTo.subn(1);
+      });
+
+      it('should revert if block.timestamp is equal to delayed until', async function () {
+        await time.increaseTo(incorrectIncreaseTo);
+        await expectRevert(
+          this.accessControl.acceptDefaultAdminTransfer({ from: correctPendingDefaultAdmin }),
+          `${errorPrefix}: delayedUntil not met`,
+        );
+      });
+
+      it('should revert if block.timestamp is less to delayed until', async function () {
+        await time.increaseTo(incorrectIncreaseTo.subn(1));
+        await expectRevert(
+          this.accessControl.acceptDefaultAdminTransfer({ from: correctPendingDefaultAdmin }),
+          `${errorPrefix}: delayedUntil not met`,
+        );
+      });
+    });
+  });
+
+  describe('cancel transfer default admin', async function () {
```

Comment:
We should add another test showing that the transfer can be cancelled after the delay.

Reply:
Also added in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4009/commits/394824665fe7bb4f15bb8b899111cff0b02a6dfe